python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py

```diff
@@ -290,10 +297,15 @@ def _swap_inputs(self, op, old_input, new_input):
                 ])
 
     def _dequantize_weights(self, graph):
+        def _is_int8_weights(op_node, weight_name):
+            weight_var_name = op_node.input(weight_name)[0]
+            weight = self._load_param(self._scope, weight_var_name)
+            return np.all(np.abs(weight % 1) < 1e-6)
```

Comment:
I guess the `1e-6` constant comes from the precision of fake-quantization. Can it be named and connected to the precision?

Reply:
The function analyzes the weights is real float32 or fake float32 (int8 range).
I have update `np.all(np.abs(weight % 1) < 1e-6)`  to `np.all(np.mod(weight, 1) == 0)` .

Reply:
@wojtuss   As the tag day is January 12 and the QA have to test the module, we merge the PR right now. If you have other comments, please let me know and I will fix in the next PR.